src/spikeinterface/qualitymetrics/tests/test_quality_metric_calculator.py

```diff
@@ -118,6 +75,7 @@ def test_compute_quality_metrics_recordingless(sorting_analyzer_simple):
     )
 
     for metric_name in metrics.columns:
+        print(metric_name)
```

Comment:
Is this helpful for test failure or was this forgotten?

Reply:
Forgotten! Resolved.